server_2x/views/layout.ejs

```diff
@@ -59,12 +59,12 @@
             </li>
             <li class="nav-item">
                 <a title="Davids super power tool" class="nav-link text-light text-center btn-sm d-none"
-                   id="menuListView" href="/mon/printerMap"><i class="fas fa-list"></i> Map</a>
+                   id="menuListView" href="/mon/map"><i class="fas fa-list"></i> Map</a>
             </li>
             <li class="nav-item">
                 <a title="Current Operations Monitoring Layout View"
                    class="nav-link text-light text-center btn-sm <%= noPrinters %>" id="menucurrentOperationsView"
-                   href="/mon/currentOp"><i class="fas fa-star"></i>
+                   href="/mon/current-ops"><i class="fas fa-star"></i>
```

Comment:
This to the changelog as well.

Reply:
Huh this should not have reached canary. Actually a good catch

Reply:
Reverted in #705